src/main/kotlin/org/jitsi/nlj/RtpReceiverImpl.kt

```diff
@@ -90,7 +90,7 @@ class RtpReceiverImpl @JvmOverloads constructor(
     private var running: Boolean = true
     private val inputTreeRoot: Node
     private val incomingPacketQueue =
-            PacketInfoQueue("rtp-receiver-incoming-packet-queue", executor, this::handleIncomingPacket)
+            PacketInfoQueue("rtp-receiver-incoming-packet-queue", executor, this::handleIncomingPacket, RtpReceiverConfig.Config.rtpReceiverQueueSizeProperty())
```

Comment:
Same not about the import style (import so we can just call `Config.rtpReceiverQueueSizeProperty()`)